pmix/borrow.py

```diff
@@ -170,97 +240,30 @@ def merge_translation_file(merge: List[str], translation_dict: TranslationDict,
-              'translations.')
-    )
-    parser.add_argument(
-        '-D', '--no_diverse', action='store_true',
-        help='If text has diverse translations, do not borrow it.'
-    )
-    parser.add_argument(
-        '-d', '--diverse',
-        help=('Supply a language. Used without the --merge argument, '
-              'this creates a worksheet that shows only strings with '
-              'diverse translations for the supplied language.')
-    )
-    parser.add_argument(
-        '-a', '--add', action='append',
-        help=('Add a language to the resulting output. The translation file '
-              'will have a column for that language. Or, the merged XLSForm '
-              'will include columns for that language and have translations '
-              'for them if possible. This option can be supplied multiple '
-              'times.')
-    )
-    parser.add_argument(
-        '-i', '--ignore', action='append',
-        help=('A language to ignore when collecting and making '
-              'translations. This option can be supplied multiple times')
-    )
-    parser.add_argument(
-        '-c', '--carry', action='store_true',
-        help=('If translations are missing, carry over the same text from '
-              'the source language. The default is to leave missing.')
-    )
-    parser.add_argument(
-        '-o', '--outfile',
-        help=('Path to write output. If this argument is not supplied, then '
-              'defaults are used. If a command would produce multiple '
-              'outputs, then do not use this argument. Instead use '
-              '"--outdir".')
-    )
-    parser.add_argument(
-        '-O', '--outdir',
-        help=('A directory to use (and create if it does not exist) for '
-              'writing output. Ignored if -o is supplied. Defaults are used'
-              'for filenames. If neither outfile nor outdir are supplied, '
-              'then default filenames are used in the current directory.')
-    )
-
     args = parser.parse_args()
-    print(args)
-    borrow(
-        xlsxfiles=args.xlsxfile,
-        correct=args.correct,
-        merge=args.merge,
-        merge_all=args.merge_all,
-        no_diverse=args.no_diverse,
-        diverse=args.diverse,
-        add=args.add,
-        ignore=args.ignore,
-        carry=args.carry,
-        outfile=args.outfile,
-        outdir=args.outdir,
-    )
+    borrow_vars: List[str] = borrow.__code__.co_varnames
+    borrow_attrs: List[str] = borrow_vars[:borrow.__code__.co_argcount]
+    kwargs = {
+        attr: getattr(args, attr)
+        for attr in borrow_attrs
+    }
+    borrow(**kwargs)
 
 
 # pylint: disable=too-many-locals
-def borrow(*, xlsxfiles: List[str], correct: List[str], merge: List[str],
-           merge_all: List[str], add: List[str], ignore: List[str],
-           no_diverse: bool = False, diverse: str = None, carry: bool = False,
-           outfile: str = None, outdir: str = None):
+def borrow(
```

Comment:
Leave `*` here. The borrow function has too many parameters that knowing the order doesn't help. I want all arguments to be keyword only.

Reply:
@jkpr Ahh, ok. That is what the `*` was for. I asked you if you knew what this was on Slack, but you must have missed it. And I couldn't find the answer by googling. Will add.